notebooks/103-paddle-onnx-to-openvino/103-paddle-onnx-to-openvino-classification.ipynb

```diff
@@ -11,22 +11,16 @@
     "\n",
```

Comment:
I got a timeout error. I know this is because the Hub Server cannot connect with paddlepaddle.org. So, I am wondering if we can include a note to say that this part (PaddlePaddle model downloading) could fault.

Reply:
Error with omz_python module on 110. 

![image](https://user-images.githubusercontent.com/10940214/155993026-c9da210b-9161-4b24-96d9-a6f77ac9952a.png)

Reply:
Question over 112 - I am expecting to have high throughput with INT8. What is the reason we have less throughput with INT8?

![image](https://user-images.githubusercontent.com/10940214/155996380-15d31a1a-2f09-4022-b629-7b4b803cfe02.png)